primers_test.go

```diff
@@ -80,3 +81,34 @@ func TestMeltingTemp(t *testing.T) {
 		t.Errorf("MeltingTemp has changed on test. Got %f instead of %f", calcTM, expectedTM)
 	}
 }
+
+func ExampleNucleobaseDeBruijnSequence() {
+	a := NucleobaseDeBruijnSequence(4)
+
+	fmt.Println(a)
+	// Output: AAAATAAAGAAACAATTAATGAATCAAGTAAGGAAGCAACTAACGAACCATATAGATACATTTATTGATTCATGTATGGATGCATCTATCGATCCAGAGACAGTTAGTGAGTCAGGTAGGGAGGCAGCTAGCGAGCCACACTTACTGACTCACGTACGGACGCACCTACCGACCCTTTTGTTTCTTGGTTGCTTCGTTCCTGTGTCTGGGTGGCTGCGTGCCTCTCGGTCGCTCCGTCCCGGGGCGGCCGCGCCCCAAA
+}
+
+func ExampleCreateBarcodes() {
+	barcodes := CreateBarcodes(20, 4, []string{"CTCTCGGTCGCTCC"}, []func(string) bool{})
```

Comment:
Passing a slice of string functions that return a bool? What's going on with this?